build.rs

```diff
@@ -71,81 +99,86 @@ fn cpu_supports_pt() -> bool {
     let ebx_out: u32;
 
     unsafe {
-        asm!(r"
-              mov $1, %eax;
-              mov $2, %ecx;
-              cpuid;"
-            : "={ebx}" (ebx_out)
-            : "i" (LEAF), "i" (SUBPAGE)
-            : "eax", "ecx", "edx"
-            : "volatile");
+        asm!(
+              "cpuid",
+              inout("eax") LEAF => _,
+              inout("ecx") SUBPAGE => _,
+              lateout("ebx") ebx_out,
+              lateout("edx") _,
+        );
```

Comment:
You could use [`core::arch::x86_64::__cpuid_count`](https://doc.rust-lang.org/stable/core/arch/x86_64/fn.__cpuid_count.html) instead.

Reply:
Nice! How did I not know about this?

Will do this in a separate PR, I think.

Thanks